wordpress-mobile/WordPress-Android
# Issue/5217 stop image upload

Fixes #5217.

I've branched off of `issue/5296-image-settings` because this PR depends on the changes there. That's why the [image settings PR](https://github.com/wordpress-mobile/WordPress-Android/pull/5709) needs to be merged first.

To test:

1. Start uploading an image
2. Tap on the image
3. Tap OK to cancel the upload
4. Notice the image is removed
